Xamarin.Forms.Platform.iOS/Cells/CellRenderer.cs

```diff
@@ -12,6 +12,9 @@ public class CellRenderer : IRegisterable
 
 		public virtual UITableViewCell GetCell(Cell item, UITableViewCell reusableCell, UITableView tv)
 		{
+			var reference = Guid.NewGuid().ToString();
```

Comment:
iOS hadn't been instrumented before. Please note that it is not fair to compare iOS results to Android results if for no other reason than the instrumentation is vastly different in many cases.